src/server/serve.rs

```diff
@@ -68,8 +69,11 @@ pub async fn serve(args: Args) -> BoxResult<()> {
 
     let server = hyper::Server::try_bind(&address)?.serve(make_svc);
     let address = server.local_addr();
-    println!("Files served on http://{}{}", address, path_prefix);
-    server.await?;
+    println!(
```

Comment:
`eprintln` probably would be better.

Reply:
I am just thinking of this. Thank for this!

Reply:
fixed ef28b18